### DISCUSSION TITLE
GPG key deleted by mistake

### DISCUSSION THREAD

**Discussion by manisha-kurukuri** [community/community#143609] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Profile

### Body

Hi everyone,

I've recently replaced my old laptop (which was formatted) with a new one. In the process, I deleted my profile GPG keys from GitHub. Now, however, I’m noticing that my past commits on GitHub repositories are showing an "unverified" status.

Is there a way to resolve this without needing access to the old laptop?

Thanks in advance for any guidance!

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @manisha-kurukuri ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by aeharding** [comment]
https://security.stackexchange.com/a/229738
